apps/e2e/helpers/cr-sqlite.ts

```diff
@@ -338,3 +346,107 @@ export async function createReconciliationOrder(db: DB, supplierOrderIds: number
 	);
 	return recondOrder[0].id;
 }
+
+export async function finalizeReconciliationOrder(db: DB, id: number) {
```

Comment:
This is duplicated from [apps/web-client/src/lib/db/cr-sqlite/order-reconciliation.ts](https://github.com/librocco/librocco/blob/45aed8e601eccd3dc0cabf38225c1d29889e6755/apps/web-client/src/lib/db/cr-sqlite/order-reconciliation.ts#L176) right? I think we should find a way to avoid the duplication (like a symlink) or at least we should leave a comment here pointing to the original function.

My concern is that in the future we will update one but not the other and will have problems because of that.

Reply:
We already had this discussion: the helpers are quite similar (but not exactly the same) as the ones used in the app. If a discrepancy happens, it will be evident as the person introducing it will:
1. Immediately be notified (by failing e2e)
2. Know where they might have introduced it (which function was edited)

☝️ my points for why the symlink wouldn't work

However, definitely couldn't hurt to add a comment at the top of the file, explaining this.